ibis/backends/clickhouse/tests/test_client.py

```diff
@@ -62,7 +62,7 @@ def test_limit_overrides_expr(con, alltypes):
 
 
 def test_limit_equals_none_no_limit(alltypes):
-    with config.option_context('sql.default_limit', 10):
+    with config.option_context('base_sqlalchemy.default_limit', 10):
```

Comment:
This is a configuration option named `sql`, not a module name, not sure if you automatically changed `sql.` by `base_sqlalchemy.`, but many changes don't seem to make sense.

Reply:
yeah, something like that, sorry for the noise!